config/initializers/datadog.rb

```diff
@@ -26,12 +26,16 @@
     c.tracing.instrument :redis, service_name: 'vets-api-redis'
     c.tracing.instrument :pg, service_name: 'vets-api-pg'
     c.tracing.instrument :http, service_name: 'vets-api-net-http'
+    c.tracing.log_injection.enabled = false
```

Comment:
I don't see the docs for this. I see `tracing.log_injection` mentioned in these docs, though https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/ruby/. Is this the correct syntax?

```suggestion
    c.tracing.log_injection = false
```

Reply:
The tracing.log_injection.enabled setting in Datadog controls whether trace IDs and span IDs are automatically added to your application logs. When enabled, this feature allows for easier correlation between logs and traces in Datadog's APM and Log Management services. This correlation is particularly useful for debugging and understanding the flow of requests through your application.

Reply:
> c.tracing.log_injection is an object, not a boolean.

Those docs I linked say it's a boolean. 
![image](https://github.com/user-attachments/assets/98c706af-d273-4a1d-84eb-806a8fab529e)

Reply:
@RachalCassity, there's an errors in dev db-migrate logs: https://argocd.vfs.va.gov/applications/vets-api-dev?operation=false
```
NoMethodError: undefined method `enabled=' for true (NoMethodError)
    c.tracing.log_injection.enabled = false
                           ^^^^^^^^^^
/app/config/initializers/datadog.rb:29:in `block in <top (required)>'
/app/config/initializers/datadog.rb:7:in `<top (required)>'
```